HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -2053,40 +2118,61 @@ def checkConditions(self,verbose=False,public_call=True):
 
         EPermShkInv=np.dot(self.PermShkDstn[0][0],1/self.PermShkDstn[0][1])
         PermGroFacAdj=self.PermGroFac[0]*EPermShkInv
-        Thorn=self.LivPrb[0]*(self.Rfree*self.DiscFac)**(1/self.CRRA)
-        GIF=Thorn/PermGroFacAdj
-        #Evaluate and report on the growth impatience condition
+        Thorn = (self.Rfree*self.DiscFac*self.LivPrb[0])**(1/self.CRRA)
```

Comment:
Is this change to thorn a bug-fix or did you redefine thorn? I'm talking about the fact that livprb was *not* exponentiated earlier.

Reply:
looking at the derivations, this seems to actually be a bugfix (the probability would be present on the rhs in eqn 10 here www.econ2.jhu.edu/people/ccarroll/public/lecturenotes/consumption/PerfForesightCRRA/ ). So maybe this should be mentioned in the release notes beyond the fact that this PR improves documentation and clarifies some important calculations